iotdb/src/main/java/org/apache/iotdb/db/query/control/QueryResourceManager.java

```diff
@@ -142,6 +142,23 @@ public void beginQueryOfGivenExpression(long jobId, IExpression expression)
     }
   }
 
+  /**
+   * Begin query and set query tokens of all filter paths in expression. This method is used in
+   * filter calculation.
+   * @param remoteDeviceIdSet device id set which can not handle locally
+   * Note : the method is for cluster
+   */
+  public void beginQueryOfGivenExpression(long jobId, IExpression expression,

```

Comment:
Does this method will replace beginQueryOfGivenExpression(long jobId, IExpression expression) in the cluster mode?

Reply:
No，in cluster module, these two methods are both used.